docs/installation.md

```diff
@@ -72,34 +72,48 @@ If you want to help develop Lychee, install the development dependencies by remo
 
 ### Configuration
 
+After installing Lychee, some additional configuration is necessary as detailed out below.
+
 #### Public Directory
```

Comment:
I wonder wether this section actually belongs here or should be moved below `## Webserver configuration`.

Reply:
> I wonder wether this section actually belongs here or should be moved below `## Webserver configuration`.

Fair point. The `public` directory stuff, anyway.

Reply:
OK, I move the `public` directory stuff.

Reply:
Fixed in https://github.com/LycheeOrg/LycheeOrg.github.io/pull/56/commits/de1be208cc49bbbcb794b7f29764122081be4c7b